From 46c4a6ceed751aff599a6a79c662b8a06164176e Mon Sep 17 00:00:00 2001
From: Ray Strode <rstrode@redhat.com>
Date: Thu, 12 Mar 2015 09:32:24 -0400
Subject: [PATCH 04/12] lib: clean up debug message

---
 src/libaccountsservice/act-user-manager.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/libaccountsservice/act-user-manager.c b/src/libaccountsservice/act-user-manager.c
index cfb3585..1e672c8 100644
--- a/src/libaccountsservice/act-user-manager.c
+++ b/src/libaccountsservice/act-user-manager.c
@@ -2117,8 +2117,6 @@ on_console_kit_session_proxy_gotten (GObject *object, GAsyncResult *result, gpoi
 static void
 get_session_proxy (ActUserManager *manager)
 {
-        g_debug ("get_session_proxy");
-
 #ifdef WITH_SYSTEMD
         if (LOGIND_RUNNING()) {
                 manager->priv->seat.state++;
@@ -2127,6 +2125,8 @@ get_session_proxy (ActUserManager *manager)
         }
 #endif
 
+        g_debug ("ActUserManager: fetching user proxy");
+
         g_assert (manager->priv->seat.session_proxy == NULL);
 
         console_kit_session_proxy_new (manager->priv->connection,
-- 
1.9.5

