From 625201106fcf376b277e923edc3f66f130e5afe8 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Jani=20V=C3=A4limaa?= <wally@mageia.org>
Date: Sat, 22 Dec 2012 17:35:01 +0200
Subject: [PATCH] Filter users with /bin/true login shell from user list

---
 src/user-classify.c | 8 +++++++-
 1 file changed, 7 insertions(+), 1 deletion(-)

--- accountsservice-0.6.37/src/user-classify.c.b4	2014-07-20 19:40:02.581611462 +0100
+++ accountsservice-0.6.37/src/user-classify.c	2014-07-20 19:41:04.784123710 +0100
@@ -80,6 +80,7 @@
 
 #define PATH_NOLOGIN "/sbin/nologin"
 #define PATH_FALSE "/bin/false"
+#define PATH_TRUE "/bin/true"
 
 #ifdef ENABLE_USER_HEURISTICS
 static gboolean
@@ -90,7 +91,7 @@
         gboolean ret = FALSE;
 
         if (shell != NULL) {
-                char *basename, *nologin_basename, *false_basename;
+                char *basename, *nologin_basename, *false_basename, *true_basename;
 
 #ifdef HAVE_GETUSERSHELL
                 char *valid_shell;
@@ -108,6 +109,7 @@
                 basename = g_path_get_basename (shell);
                 nologin_basename = g_path_get_basename (PATH_NOLOGIN);
                 false_basename = g_path_get_basename (PATH_FALSE);
+                true_basename = g_path_get_basename (PATH_TRUE);
 
                 if (shell[0] == '\0') {
                         ret = TRUE;
@@ -115,11 +117,14 @@
                         ret = TRUE;
                 } else if (g_strcmp0 (basename, false_basename) == 0) {
                         ret = TRUE;
+                } else if (g_strcmp0 (basename, true_basename) == 0) {
+                        ret = TRUE;
                 }
 
                 g_free (basename);
                 g_free (nologin_basename);
                 g_free (false_basename);
+                g_free (true_basename);
         }
 
         if (password_hash != NULL) {
